upload/system/library/emshelper.php

```diff
@@ -637,7 +630,7 @@ public function getOrderLineTaxRate($paymentMethod, $price, $taxClassId)
             }
         }
 
-        return static::formatAmountToCents($taxRate);
+        return (int) round($taxRate * 100);
```

Comment:
Why is it not taking into account currency (without using the getAmountInCents function)?

Reply:
Because formatAmountToCents actually working incorrect and return mismatched value